sonar-client/lib/client.js

```diff
@@ -203,6 +203,15 @@ module.exports = class SonarClient {
     })
   }
 
+  async change (config, key) {
```

Comment:
I think this needs a context, `client.change()` is too ambigous. Maybe `client.updateIsland()` or `changeIsland`? We didn't really use `change` in methods much. Don't really mind too much I think. I'd likely go with `updateIsland`.

Reply:
renamed function to updateIsland